### PR TITLE
WNP 118 (#13619)

### LINKED ISSUE #13619: WNP 118 EU (translate)
## Description

Copy doc: https://docs.google.com/document/d/1mCJtM_yeiZt5Wdo09eICOdRXhRgScvmiQxLy2io-LYU/edit?usp=sharing
Figma file: https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/EU_Firefox_WNP_21-23?type=design&node-id=3396-26&mode=design&t=XrOWgiBLD1lqoR8r-0

Note:

The page has two CTAs based on FF being default or not:

- "Set Firefox as your default browser"
- "Try it now" (which will link to the homepage in a different language)

---

#### Success Criteria
- [ ] Live before Sept 26

### PR DESCRIPTION
**Pull request by stephaniehobson**
Add WNP for 118.

## Significant changes and points to review

- add EU pages: en-GB, FR, DE
- add NA pages: en-US and en-CA

## Issue / Bugzilla link

Fix #13619

## Testing

Copy doc and Figma file are[ linked in the issue](https://github.com/mozilla/bedrock/issues/13619).

EU:
- show a button to set Fx as default if it is not default, otherwise show a link to try it now
https://www-demo2.allizom.org/fr/firefox/118.0/whatsnew/
https://www-demo2.allizom.org/de/firefox/118.0/whatsnew/
https://www-demo2.allizom.org/en-GB/firefox/118.0/whatsnew/

NA:
- show a link to try it now
https://www-demo2.allizom.org/en-US/firefox/118.0/whatsnew/
https://www-demo2.allizom.org/en-CA/firefox/118.0/whatsnew/

### CODE REVIEW

File: tests/pages/firefox/whatsnew/whatsnew_118.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    _set_alt_cta_button_locator = (By.CSS_SELECTOR, "#cta-try .mzp-c-button")
+    # NA pages
+    _set_try_it_button_locator = (By.ID, "cta-now")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+        el = self.find_element(By.TAG_NAME, "html")
+        self.wait.until(lambda s: "loaded" in el.get_attribute("class"))
+        # EU pages UI Tour call will time out at 1 second, NA pages don't have UI Tour call
+        sleep(1.2)
```
Comment (alexgibson) [inline tests/pages/firefox/whatsnew/whatsnew_118.py:25]: We should try and avoid doing implicit waits in our tests, as these can create flaky results. Is there any way you can change this to an explicitly wait on some kind, like a class name that gets added?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx118-en-CA.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        </aside>
+
+        <h2>Say ‘<span lang="fr">Bonjour</span>’ to Firefox&nbsp;translations</h2>
+
+        <p>Translate the web within Firefox, so your data stays safe.</p>
+
+        <p>Click the {% include 'firefox/whatsnew/includes/fx118/translate-icon.svg' %} icon in Firefox’s address bar and enjoy content across the web.</p>
+
+        <p>
+          <a id="cta-now" class="mzp-c-button mzp-t-product" href="https://www.mozilla.org/es-MX/" data-cta-type="button" data-cta-text="Try it now">Try it now</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx118-en-CA.html:44]: Seeing as the header says bonjor, should this CTA link to the French home page?
```suggestion
          <a id="cta-now" class="mzp-c-button mzp-t-product" href="https://www.mozilla.org/fr/" data-cta-type="button" data-cta-text="Try it now">Try it now</a>
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
Only a couple of small things, r+wc nice work!

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx118-en-CA.html (hunk 2 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx118-en-CA.html:44]: Yes, thank you.

**Comment by stephaniehobson** [comment]
Updated. I separated the tests into two separate files so I could wait on a class and remove the sleep.

**Pull request merged**